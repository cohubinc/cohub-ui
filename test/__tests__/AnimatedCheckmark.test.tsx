import React from "react";
import { mount } from "enzyme";

import { AnimatedCheckmark } from "dist";

describe("AnimatedCheckmark", () => {
  it("renders without crashing", async () => {
    const wrapper = mount(<AnimatedCheckmark color="currentcolor" />);

    expect(wrapper).toMatchSnapshot();
  });
});
