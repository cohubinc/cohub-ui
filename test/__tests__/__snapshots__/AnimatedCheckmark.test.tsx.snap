// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedCheckmark renders without crashing 1`] = `
<AnimatedCheckmark
  color="currentcolor"
>
  <div
    style={
      Object {
        "color": "currentcolor",
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <svg
      className="AnimatedCheckmark"
      viewBox="0 0 52 52"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        className="checkmark__circle"
        cx="52"
        cy="52"
        fill="none"
        r="1000"
      />
      <path
        className="checkmark__check"
        d="M14.1 27.2l7.1 7.2 16.7-16.8"
        fill="none"
      />
    </svg>
  </div>
</AnimatedCheckmark>
`;
